lib/fs.js

```diff
@@ -2138,3 +2138,18 @@ SyncWriteStream.prototype.destroy = function() {
 };
 
 SyncWriteStream.prototype.destroySoon = SyncWriteStream.prototype.destroy;
+
+fs.mkdtemp = function(template, callback) {
+  if (typeof callback !== 'function') {
```

Comment:
Any reason why this method raises a TypeError here rather than using `maybeCallback`?